<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<title>Nicholas B Pratt</title>
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link href="https://fonts.googleapis.com/css?family=Inter:400,700&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="style.css">
</head>

<body>
	<nav class="navbar">
		<div class="nav-container">
			<div class="nav-brand">Nicholas B. Pratt</div>
			<ul class="nav-links">
				<li><a href="#home" class="active">Home</a></li>
				<li><a href="#portfolio">Portfolio</a></li>
				<li><a href="#cv">CV</a></li>
			</ul>
		</div>
	</nav>
	<main class="container">
		<section class="header">
			<h1>About Me</h1>
			<div class="about-flex">
				<div class="about-img">
					<img src="Nicholas_Pratt_Original.jpg" height="180" width="260">
				</div>
				<div class="about-text">	
					<p class="lead">
						Hello, my name is Nicholas Bane Pratt, and I study genomes - how they vary, function, and how they can be modeled using various tools under the 'machine learning' umbrella. I am currently an undergraduate Computational Biology & Bioinformatics researcher at the University at Buffalo, New York's flagship public research institution. Currently I am working under Dr. Victor Albert and Dr. Jaroslaw Zola, and I also collaborate with and receive mentorship from Dr. Michael Gore at Cornell University. 
						
						There are many problems facing the physical world - many for which there is no clear path forward to a solution. It is my conviction that as we decode how gene sequences interact throughout the central dogma to create structures, phenotypes, traits. I believe solutions to energy production, climate change, disease, food security, and beyond will emerge. These answers all lie downstream of solving the genome. 
						
						When I talk about “solving the genome,” I mean developing a foundational understanding of how DNA sequences functionally interact to create life and the world around us. Perhaps the best way to describe this is to imagine what such a future would look like.
						
						Having full knowledge of how genomes work at single nucleotide resolution implies that we can functionally predict the downstream effect altering even a single base would have on an organism. With this, engineering any already known trait from one organism into another becomes trivial. Imagine a world with citrus trees as cold-hardy as apple trees, humans with the lifespans of bowhead whales, and wild animal microbial communities can be engineered to digest plastic waste - just as the waxworm’s microbiome already can. We might one day grow raw meat from a tomato plant or write entirely synthetic genomes that produce emergent, never-before-seen traits. Imagine now living symbiotic clothing, roads made of a callus-like material that 'heals' its potholes, and other advancements in 'living' architecture and materials science. Here, the limit is one's imagination. 
						
						I believe that working toward this future of solving the genome, at this point, is the most important thing humanity can do. I’m interested in tackling the technical, conceptual, and experimental problems that stand in the way. This includes everything from traditional Mendelian breeding experiments, to the quantitative genomic methods widely used today, to cutting-edge applications of neural networks in genome modeling. I have informal training in the former and formal experience in the latter two.
						
						In my current lab, I use Gaussian Process and Random Forest regression to model daily stomatal conductance in a diverse panel of maize genotypes. The goal of this work is to inform selection decisions that result in more drought-tolerant crop varieties. This expands foundational understanding of genomics and provides insight into how slight genotypic variation affects quantitative traits, in this case, timing and magnitude of stomatal conductance. 
						
						One of the core projects I led was a proof-of-concept study to conduct transfer learning on Oxford Nanopore’s open-source ML model, Remora. These basecalling models are generally trained to recognize only canonical nucleotides, but I retrained Remora so that it could also detect 8-oxo-guanine, which is a common marker of oxidative DNA damage and cancer, in raw signal sequencing data. I then integrated this into Dorado, Nanopore’s convolutional neural network basecaller. This work was recognized by the Barry Goldwater Foundation with a 2025 scholarship.
						
						Across my research, I work at the forefront of various genomics fields; from quantitative genetics to next generation sequencing and predictive genome modeling. All this has reinforced my fervor to work on solving the genome, which I believe will catapult civilization into a new technological revolution. 
						
						Outside of lab, I am a project manager at a construction company in Buffalo where I bid on projects and develop new relationships with clients. I also enjoy gardening and have been working to restore a former dump site to native habitat. I paint, primarily watercolor landscapes and mixed-media abstractions. I self study contemporary philosophy, of which René Girard's ideas on mimetic desire and scapegoating seem particularly prescient today. I study Central and Eastern European history, culture, and current geopolitics both as a hobby and at university where I studied abroad in Czechia as a U.S. Department of Defense Gilman Scholar. I love being out in the wilderness where I primarily enjoy camping, swimming in the great lakes and New York's many gorges and waterfalls, and hiking with the goal of reaching the highest point in each of the 50 states. As of July 2025, my standing is 9/50.
					</p>
				</div>
			</div>
		</section>
		<section class="header">
			<h2>Contact</h2>
			<p>
				Nicholas B. Pratt<br>
				University at Buffalo<br>
				Email: <a href="mailto:your@email.edu">nbpratt@buffalo.edu</a><br>
			</p>
		</section>
		
		<footer class="footer">
			<p>&copy; 2025 Nicholas B. Pratt</p>
		</footer>
	</main>
</body>
</html>
